janusgraph-core/src/main/java/org/janusgraph/graphdb/query/vertex/SimpleVertexQueryProcessor.java

```diff
@@ -118,12 +118,7 @@ public Long apply(@Nullable Entry entry) {
      * @return
      */
     private Iterator<Entry> getBasicIterator() {
-        EntryList result = vertex.loadRelations(sliceQuery, new Retriever<SliceQuery, EntryList>() {
-            @Override
-            public EntryList get(SliceQuery query) {
-                return QueryProfiler.profile(profiler,query, q -> tx.getGraph().edgeQuery(vertex.longId(), q, tx.getTxHandle()));
-            }
-        });
+        final EntryList result = vertex.loadRelations(sliceQuery, query -> QueryProfiler.profile(profiler,query, q -> tx.getGraph().edgeQuery(vertex.longId(), q, tx.getTxHandle())));
```

Comment:
Add space between profiler and query args for consistency.

Reply:
done